### ISSUE TITLE
After switching to Grunt for tests, compilation no longer happens before testing.

### ISSUE THREAD

**Issue by Mark-Simulacrum** [MiguelCastillo/amd-resolver#19]
The gulp version of the tests ran the compilation step before running tests. Grunt does not do this.

Solutions:
 - Fix Gulp running tests so that tests work correctly.
 - Make Grunt run gulp compilation task.

**Comment by MiguelCastillo** [comment]
Yeah I had to switch back to grunt because the mocha-phantomjs integration in gulp is really shitty... XHR does not work.  Once that's fixed, we can go back to an all gulp workflow again.
